src/main/scala/org/ergoplatform/settings/ValidationRules.scala

```diff
@@ -140,7 +140,7 @@ object ValidationRules {
     hdrVotesUnknown -> RuleStatus(s => fatal(s"First header of an epoch should not contain a vote for unknown parameter. $s"),
       Seq(classOf[Header]),
       mayBeDisabled = true),
-    hdrCheckpointV2 -> RuleStatus(s => fatal(s"First version 2 header on the mainnet on height 417,729 should have " +
+    hdrCheckpoint -> RuleStatus(s => fatal(s"First version 2 header on the mainnet on height 417,729 should have " +
```

Comment:
Should the hard-coded height be changed to the one set in the config file?

Reply:
Changed comment